### ISSUE TITLE
Align palette colours to WCAG Level AA

### ISSUE THREAD

**Issue by sgoudham** [catppuccin/catppuccin#2588]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### What is your concern?

At the time of writing, some/majority of the colour combinations in the v0.2.0 palette are not WCAG AA compliant and fail to offer sufficient colour contrast. This is most noticeable in the **Latte** flavour.

If we reach such a position where we are comfortable revisiting the colours in the palette, we should ensure that our colour combinations offer more colour contrast. However, that being said, it is worth mentioning that we have had internal experiments with improving the colour contrast which, for lack of a better term, resulting in the colour palette looking more "dull" and not very "pastel". Fundamentally, I think there's a tradeoff to be made as I, currently, don't think a fully WCAG Level AA compliant palette will keep the spirit and feel of a "soothing pastel theme." 

To be clear, revisiting the existing colours is not something the staff/core teams are looking into at the minute as we're more focused and interested on making sure the organisation is more maintainable with the ongoing rollout of [catppuccin/whiskers](https://github.com/catppuccin/whiskers).

**Comment by Crazycrystaldev** [comment]
Can I work for this as contributor?

**Comment by backwardspy** [comment]
> Can I work for this as contributor?

thanks for your interest! we're not currently pursuing a palette update; as per the last paragraph of @sgoudham's original post this is more a tracking issue for potential future improvements.

our primary goal right now is to get whiskers into as many ports as possible. we want to make any future palette updates like this one *much easier* to roll out. if you're looking to make a contribution, perhaps consider checking out out the list of ports not using whiskers: https://arewewhiskersyet.com

bear in mind that whiskers is not fit for use on every port. if its implementation seems difficult, or you're unsure whether it's a good fit, then feel free to raise an issue on the port or message in the dev channel on discord! we'll do our best to help.
